### PR TITLE
OP-599 extend pex note field

### PR DESCRIPTION
**Pull request by mwithi**
See OP-599.

Cc: @claudiorosazza

### CODE REVIEW

File: src/main/java/org/isf/examination/manager/ExaminationBrowserManager.java [hunk 1]
```diff
@@ -299,9 +299,9 @@ public String getDiuresisDescriptionKey(String description) {
 	 */
 	protected List<OHExceptionMessage> validateExamination(PatientExamination patex) {
 		List<OHExceptionMessage> errors = new ArrayList<>();
-		if (patex.getPex_note().length() > 300) {
+		if (patex.getPex_note().length() > PatientExamination.PEX_NOTE_LENGHT) {
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/examination/manager/ExaminationBrowserManager.java:302]: This variable is spelled wrong; `LENGHT` should be `LENGTH`.

File: src/main/java/org/isf/examination/model/PatientExamination.java [hunk 2]
```diff
@@ -44,10 +48,19 @@
...
+    @AttributeOverride(name="createdDate", column=@Column(name="PEX_CREATED_DATE")),
+    @AttributeOverride(name="lastModifiedBy", column=@Column(name="PEX_LAST_MODIFIED_BY")),
+    @AttributeOverride(name="active", column=@Column(name="PEX_ACTIVE")),
+    @AttributeOverride(name="lastModifiedDate", column=@Column(name="PEX_LAST_MODIFIED_DATE"))
+})
 public class PatientExamination implements Serializable, Comparable<PatientExamination> {
 
 	private static final long serialVersionUID = 1L;
-
+	public static final int PEX_NOTE_LENGHT = 2000;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/examination/model/PatientExamination.java:62]: See note above; and when this is fixed the GUI PR needs to be updated too.

**Comment by dbmalkovsky** [comment]
Other than the comments it LGTM.

File: src/main/java/org/isf/examination/manager/ExaminationBrowserManager.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/examination/manager/ExaminationBrowserManager.java]: Thanks!

**Pull request merged**